CodeReview/src/CodeReviewManager.cpp

```diff
@@ -88,6 +89,8 @@ void CodeReviewManager::saveReview(QString newVersion)
 NodeReviewsList* CodeReviewManager::loadReview(QString newVersion, VersionControlUI::DiffSetup& diffSetup,
 																 Visualization::ViewItem* viewItem)
 {
+	CFocus::clearFocusInformation();
```

Comment:
At the moment it is needed there, but I'll try to improve the CFocus command in the next iteration and move this stuff as well.